### ISSUE TITLE
Will you accept I to scan them into text and upload?

### ISSUE THREAD

**Issue by hubred-rev** [ProletRevDicta/Prolet#5]
You said you need **MEGA**, but **GitHub** can register by **Proton-mail (A secret mail)**, so I think this is unnecessary. My mail is **redgit@protonmail.com**. And if you want talk more with me, please via **Proton-mail (This is very safe)**.
**All scan costs by me.**
This is the scan and generate codes:
https://github.com/hubred-rev/codes

**Comment by hubred-rev** [comment]
Even more, we can make a wiki(html) by these text files.

**Comment by ProletRevDicta** [comment]
I agree with you.  A wiki website is necessary for us. Maybe we should contact with the BANNEDTHOUGHT net？

**Comment by DiazoPlus** [comment]
MEGA can also register by Proton-mail.
There are some binary files that cannot be converted to text files, such as photos and videos, exceeding the file size limit on github, I will upload it to mega.
---
wiki website may be helpful. To have a more detailed discussion, you can email to me. my email is jcQYCGTjU7Tu@protonmail.com

**Comment by hubred-rev** [comment]
> I agree with you. A wiki website is necessary for us. Maybe we should contact with the BANNEDTHOUGHT net？

Contacted.

**Comment by ProletRevDicta** [comment]
My mail is ProletRevDicta@protonmail.com

**Comment by DiazoPlus** [comment]
> 
> 
> My mail is [ProletRevDicta@protonmail.com](mailto:ProletRevDicta@protonmail.com)

you should not expose this email address. please delete this issue
